apps/frontend/src/app/Components/Character/CharacterCardPico.tsx

```diff
@@ -30,7 +30,7 @@ export default function CharacterCardPico({ characterKey, onClick, simpleTooltip
   )
 
   const simpleTooltipWrapperFunc = useCallback((children: ReactNode) =>
-    <BootstrapTooltip placement="top" title={

```

Comment:
this variant should have no deplay, since its just a simple name tooltip that goes in the main tooltip